src/core/lib/security/authorization/grpc_authorization_engine.h

```diff
@@ -56,6 +56,12 @@ class GrpcAuthorizationEngine : public AuthorizationEngine {
   // Required only for testing purpose.
   size_t num_policies() const { return policies_.size(); }
 
+  // Required only for testing purpose.
+  Rbac::AuditCondition audit_condition() const { return audit_condition_; }
+
+  // Required only for testing purpose.
+  size_t num_audit_loggers() const { return audit_loggers_.size(); }
```

Comment:
I added test with custom loggers to verify the logger config. IIUC, it's not very useful to expose loggers here because built-in loggers (just stdout logger for now) don't normally retain configs, unless we make them to in tests. But that can be done via the custom factory as I just did. PTAL.

Reply:
Ah, right, I somehow thought this object was storing the configs, not the actual logger instances.

The change you made it useful, but let's also do this: Add a `name()` method to the logger itself, and then expose the list of loggers here.  That way, we can also verify the order of the loggers, at least in cases where there are not two instances of the same logger type.